Commit ID: 601338ce158814652d5d2fbd766c62864ef87ec1
Change ID: ktmxnorpplounvqvqqkmpsywoptoosrz
Author   : Gabriel Fontes <hi@m7.rs> (2025-12-18 09:23:01)
Committer: Gabriel Fontes <hi@m7.rs> (2025-12-18 14:31:41)
Signature: (no signature)

    feat(tasklist): tweak completed and progressbar colors

diff --git a/plugins/libkolab/skins/elastic/include/darkmode.less b/plugins/libkolab/skins/elastic/include/darkmode.less
index febed080ae..d4d9f382f6 100644
--- a/plugins/libkolab/skins/elastic/include/darkmode.less
+++ b/plugins/libkolab/skins/elastic/include/darkmode.less
@@ -147,7 +147,7 @@
     }
 
     #tasklist .progressbar .progressvalue {
-        border-color: @color-dark-warning;
+        border-color: @color-dark-main;
     }
 
     .listing {
diff --git a/plugins/libkolab/skins/elastic/include/tasklist.less b/plugins/libkolab/skins/elastic/include/tasklist.less
index 8cc8196d8d..306e13d509 100644
--- a/plugins/libkolab/skins/elastic/include/tasklist.less
+++ b/plugins/libkolab/skins/elastic/include/tasklist.less
@@ -74,7 +74,7 @@
 
             &.complete {
                 span.title {
-                    color: @color-list-deleted;
+                    color: @color-list-secondary;
                 }
 
                 .progressbar {
@@ -211,7 +211,7 @@
         right: .2em;
 
         .progressvalue {
-            border-top: 2px solid @color-warning;
+            border-top: 2px solid @color-main;
         }
     }
 
@@ -282,7 +282,7 @@
     }
 
     &.status-completed h2 {
-        color: @color-list-deleted;
+        color: @color-list-secondary;
     }
 
     &.status-flagged h2:before {
